docs/changelog.rst

```diff
@@ -13,7 +13,7 @@ Features - 20.16.0
 - Drop support for running under Python 2 (still can generate Python 2 environments) - by :user:`gaborbernat`. (`#2382 <https://github.com/pypa/virtualenv/issues/2382>`_)
 - Upgrade embedded pip to ``22.2`` from ``22.1.2`` and  setuptools to ``63.2.0`` from ``62.6.0`` -
   by :user:`gaborbernat`. (`#2383 <https://github.com/pypa/virtualenv/issues/2383>`_)
-
+- Update Nushell activation scripts to version 0.66 - by :user:`kubouch`.
```

Comment:
Not here... https://virtualenv.pypa.io/en/latest/development.html#news-entries

Reply:
I hope I did it correctly this time.